Xamarin.Forms.Core/DesignMode.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Xamarin.Forms
+{
+	public static class DesignMode
+	{
+		public static bool DesignModeEnabled { get; internal set; }
```

Comment:
We usually use `IsDesignModeEnabled`, or `IsInDesignMode` for states. e.g. `Switch.IsToggled` indicates the state, while `Switch.Toggled` is the event.